core/src/main/java/com/linecorp/armeria/common/encoding/StreamDecoderFactory.java

```diff
@@ -40,6 +40,13 @@ static StreamDecoderFactory gzip() {
         return StreamDecoderFactories.GZIP;
     }
 
+    /**
+     * Returns the {@link StreamDecoderFactory} for {@code "br"} content encoding.
+     */
+    static StreamDecoderFactory br() {
+        return StreamDecoderFactories.BR;
```

Comment:
`brotli()` and `BROTLI`? `br` seems too short to me.
```suggestion
    static StreamDecoderFactory brotli() {
        return StreamDecoderFactories.BROTLI;
```

Reply:
+1